sidebars.js

```diff
@@ -217,52 +462,136 @@ const sidebars = {
+                  label: 'Collect Logs from ScalarDL on a Kubernetes Cluster',
+                },
               ],
             },
             {
               type: 'category',
               label: 'Backup and Restore Guides',
               collapsible: true,
               items: [
-                'scalar-kubernetes/BackupRestoreGuide',
-                'scalar-kubernetes/BackupRDB',
-                'scalar-kubernetes/BackupNoSQL',
-                'scalar-kubernetes/RestoreDatabase',
-                'scalar-kubernetes/RegularCheck',
+                {
+                  type: 'doc',
+                  id: 'scalar-kubernetes/BackupRestoreGuide',
+                  label: 'Back Up and Restore ScalarDL Data in Kubernetes',
+                },
+                {
+                  type: 'doc',
+                  id: 'scalar-kubernetes/BackupRDB',
+                  label: 'Back Up an RDB in Kubernetes',
+                },
+                {
+                  type: 'doc',
+                  id: 'scalar-kubernetes/BackupNoSQL',
+                  label: 'Back Up a NoSQL Database in Kubernetes',
+                },
+                {
+                  type: 'doc',
+                  id: 'scalar-kubernetes/RestoreDatabase',
+                  label: 'Restore Databases in Kubernetes',
+                },
+                {
+                  type: 'doc',
+                  id: 'scalar-kubernetes/RegularCheck',
+                  label: 'Components to Regularly Check When Running on Kubernetes',
+                },
               ],
             },
           ],
         },
-        'backup-restore',
-        'scalar-kubernetes/HowToScaleScalarDL',
-        'scalar-kubernetes/HowToUpgradeScalarDL',
-        'scalar-manager/overview',
+        {
+          type: 'doc',
+          id: 'backup-restore',
+          label: 'Backup and Restore Data in ScalarDL',
+        },
+        {
+          type: 'doc',
+          id: 'scalar-kubernetes/HowToScaleScalarDL',
+          label: 'Scale',
+        },
+        {
+          type: 'doc',
+          id: 'scalar-kubernetes/HowToUpgradeScalarDL',
+          label: 'Upgrade',
+        },
+        {
+          type: 'doc',
+          id: 'scalar-manager/overview',
+          label: 'Scalar Manager Overview',
+        },
       ],
     },
     {
       type: 'category',
       label: 'Reference',
       collapsible: true,
       items: [
-        'compatibility',
-        'scalardl-command-reference',
-        'generic-contracts-reference',
-        'scalardl-benchmarks/README',
+        {
+          type: 'doc',
+          id: 'compatibility',
+          label: 'ScalarDL Compatibility Matrix',
+        },
+        {
+          type: 'doc',
+          id: 'scalardl-command-reference',
+          label: 'ScalarDL Client Command',
+        },
+        {
+          type: 'doc',
+          id: 'generic-contracts-reference',
+          label: '汎用コントラクトおよびファンクション',
```

Comment:
I think we don't treat `Generic contracts` as a proper noun. So, I think it's fine as is.
Maybe, it's better to be just `汎用コントラクト`.

Reply:
@feeblefakie  
Ah, sorry. My comment was a bit ambiguous.

Here is the `docsEnglish` section. Not the `docsJapanese` section.

So, simply we should use English here. I think it was translated by accident.

According to your suggestion, it might be better to use `Generic Contracts` (without `Reference`) here though.